presto-main/src/main/java/com/facebook/presto/server/PrestoSystemRequirements.java

```diff
@@ -55,8 +55,8 @@ public static void verifyJvmRequirements()
         String osName = StandardSystemProperty.OS_NAME.value();
         String osArch = StandardSystemProperty.OS_ARCH.value();
         if ("Linux".equals(osName)) {
-            if (!"amd64".equals(osArch)) {
-                failRequirement("Presto requires x86-64 or amd64 on Linux (found %s)", osArch);
+            if (!"amd64".equals(osArch) && !"ppc64le".equals(osArch)) {
```

Comment:
I suggest we warn on PPC that it is not tested.

Reply:
like `log.warn("Support for PPC is experimental"); // TODO add CI` ?